src/textual/widgets/_progress_bar.py

```diff
@@ -336,6 +404,8 @@ def key_space(self):
         self.show_eta = show_eta
 
         self.total = total
+        self._colorscheme = colorscheme
+        self._thickness = thickness
```

Comment:
Isn't it enough for those to be reactive only in Bar class and be normal variables in ProgressBar?

Reply:
If they're normal variables in the progress bar, then if someone does something like `progress_bar.thickness = 2`, that won't updated automatically the thickness of the progress bar because Textual has no way of knowing that it should also update the thickness of the bar renderable.